http/python/requests/http/requests/middleware/options/retry_handler_option.py

```diff
@@ -0,0 +1,44 @@
+from kiota.abstraction.request_option import RequestOption
+
+class RedirectHandlerOption(RequestOption):
```

Comment:
the class name is wrong, should be retry

Reply:
Resolved in https://github.com/microsoft/kiota/pull/1251/commits/fae7353bdbd8673e475fb10dc4f9503ccaf95ee8